ci/pipelines/concourse-pypi-resource-pipeline-template.yml

```diff
@@ -200,4 +201,3 @@ jobs:
   - put: pypi.docker
     params:
       image: pypi.dev-docker/image.tar
-      additional_tags: ((docker_tag_add_file))
```

Comment:
why? you want only latest? and each time to overwrite the latest?
isn't it better to use the version? you just forgot to stream the version data.

Reply:
add `inputs: [version]`